src/parser/md.rs

```diff
@@ -484,44 +395,20 @@ impl MdParser {
 
         if rest.starts_with(&self.macro_start) {
```

Comment:
Another good candidate for `strip_prefix` I guess.

Reply:
And even to combine it with `strip_suffix` 😉